config/default.yml

```diff
@@ -636,6 +636,16 @@ Style/StringLiteralsInInterpolation:
     - single_quotes
     - double_quotes
 
+Style/StringMethods:
```

Comment:
Will do. Just noticed that `CollectionMethods` is not in there - is that intentional?